dumper/app/src/test/java/com/google/edwmigration/dumper/application/dumper/ConnectorArgumentsTest.java

```diff
@@ -61,11 +62,33 @@ public void getDatabases_trimDatabaseNames() throws IOException {
   @Test
   public void getDatabases_trimDatabaseNamesFilteringOutBlankStrings() throws IOException {
     ConnectorArguments arguments =
-        new ConnectorArguments(
-            new String[] {"--connector", "teradata", "--database", "db1, ,,, db2 "});
+        new ConnectorArguments("--connector", "teradata", "--database", "db1, ,,, db2 ");
 
     List<String> databaseNames = arguments.getDatabases();
 
     assertEquals(ImmutableList.of("db1", "db2"), databaseNames);
   }
+
+  @Test
+  public void testNoUserFlag() throws IOException {
```

Comment:
I would suggest following the naming convention `<methodUnderTest>_<testCaseDescription>_<outcome>`.

```
getUserOrFail_noUserFlag_throwsException
```

Reply:
Done